### PR TITLE
fix/converstion and mac deep link

### PR DESCRIPTION
**Pull request by kf-rns**
## Description
Converted demons with fraction from settings.json file. Also redirected on click of buy on the basis of user's platform.
Fixes #
- Redirected on the basis of user platform on press of buy button
- Converted demons according to fraction in settings.json 

## Checklist
- [ ] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Run `meteor npm run lint`
- [ ] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: default_settings.json [hunk 1]
```diff
@@ -20,17 +20,32 @@
...
                 "fraction": 1000000
             },
             {
-                "denom": "pylon",
-                "displayName": "PYLON",
+                "denom": "ustripeusd",
+                "displayName": "STRIPEUSD",
+                "fraction": 1000000
+            },
+            {
```
Comment (BlackPanthar) [inline default_settings.json:39]: Do we need ujuno here as well? Is it to aid your own local config?

**Review by mustafapylons — APPROVED** [review]
Looks good to me.

File: default_settings.json (hunk 1 above)
Comment (faisalnaveedRNS) [reply default_settings.json:39]: this is configurable we can edit as many as in live server, this is just a sample

Comment (faisalnaveedRNS) [reply default_settings.json:39]: > Do we need ujuno here as well? Is it to aid your own local config?

this is configurable we can edit as many as in live server, this is just a sample

**Review by BlackPanthar — APPROVED** [review]

**Review by MikeSofaer — APPROVED** [review]

**Pull request merged**